sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -975,18 +990,20 @@ public PDone apply(PCollection<TableRow> input) {
         String tempLocation = options.getTempLocation();
```

Comment:
I want to defer it until it is needed. I am debating in general whether we want one tempLocation per storage or per use case.

I think if one tempLocation use case supports more than one storages, it need to have its own option.